freedomofpress/securedrop-client
# Fix segfault and update PyQt in dev environment

# Description

Fixes #628 .

# Test Plan

This test requires Qubes, as it cannot be reproduced in the dev environment.
- `make all` on branch https://github.com/freedomofpress/securedrop-workstation/tree/345-debian-buster
- submit something containing `>` character to source interface (e.g: `<hello>`)
- Open source in client
- [ ] Observe segfault/client crash
- Create deb of client on this branch
- [ ] The client no longer segfaults when opening the source created above

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
